test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require "test_helper"
+
+class PagesControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = sign_in users(:bob)
```

Comment:
Sorry nit pick but `@user` isn't being used yeah?

Reply:
Apologies - I was on Minitest autopilot and did what I'm used to. Should be sorted now.